domains/DataCollection/FormGallery/index.js

```diff
@@ -141,7 +140,7 @@ const FormGallery = ({
           />
         </View>
       </View>
-      {customForms.length > 0 && (
+      {customForms && (
```

Comment:
`customForms` should be an empty array if there are no forms for the organization stored in the cache. The section should still show up if there's an empty array